code/datums/components/plumbing/_plumbing.dm

```diff
@@ -29,16 +29,13 @@
 	RegisterSignal(parent, list(COMSIG_MOVABLE_MOVED,COMSIG_PARENT_PREQDELETED), .proc/disable)
 	RegisterSignal(parent, list(COMSIG_OBJ_DEFAULT_UNFASTEN_WRENCH), .proc/toggle_active)
 	RegisterSignal(parent, list(COMSIG_OBJ_HIDE), .proc/hide)
-	RegisterSignal(parent, list(COMSIG_ATOM_UPDATE_OVERLAYS), .proc/create_overlays)
+	RegisterSignal(parent, list(COMSIG_ATOM_UPDATE_OVERLAYS), .proc/create_overlays) //create overlays also gets called after init (no idea by what it just happens)
```

Comment:
Leave a `stack_trace()` to figure out if you don't know by what, and test it.
Something like this:
```
/path_of_the_object_where_this_happens/update_overlays()
  . = ..()
  stack_trace("I was called!")
```